test/rack/prx_auth_test.rb

```diff
@@ -59,15 +61,47 @@
     end
   end
 
-  describe '#token_expired?' do
-    it 'returns true if token is expired' do
-      claims['iat'] = Time.now.to_i - 4000
+  describe '#expired?' do
 
-      assert prxauth.send(:expired?, claims) == true
+    def expired?(claims)
+      prxauth.send(:expired?, claims)
     end
 
-    it 'returns false if it is valid' do
-      assert prxauth.send(:expired?, claims) == false
+    it 'is expired if the token is issued in the future' do
+      claims['iat'] = Time.now.to_i + 3600
```

Comment:
Is this a change in functionality?  I _think_ no ... but double checking.

I guess in any case, if ID is the one issuing and checking this, shouldn't run into clock/time issues between servers.

Reply:
Was also re-reading the [JWT RFC](https://tools.ietf.org/html/rfc7519#section-4.1.4) and saw the bit about clock skew ... think we need to account for that here?  Or just in ID?

Reply:
Also in the RFC is this [not before claim](https://tools.ietf.org/html/rfc7519#section-4.1.5)...

Makes me wonder if this is what we actually want here?  Is this abusing `iat` to fulfill the actual purpose of the `nbf` claim?  The RFC seems to think `iat` is just for determining the age of the token, not checking validity?

Reply:
It looks right to me, I will change this. I was concerned about a case where the iat is mistakenly at some point in the future and the exp is nowish, which would make an extremely long lived token because of the way I determine whether it's an old-style or fixed exp, but it's definitely not necessary unless there are tokens out there that I don't know about.

Reply:
+1 specs!  Yeah, I think the old/fixed-exp tokens are going to be the only ones out there.